example_config.yml

```diff
@@ -0,0 +1,23 @@
+squish_paths:
+  - '/Volumes/storage/videos/sports'
+  - '/Volumes/storage/videos/movies'
+  - '/Volumes/storage/videos/tv.shows/'
+rename_ts_paths:
+  - '/Volumes/storage/videos/tv.shows/'
+  - '/Volumes/storage/videos/sports'
+log_loc: "logs"
+processed_loc: "processed"
```

Comment:
Why not specify this like the other paths  `/Volumes/storage/videos/processed`. And why the double quotes?

Reply:
Double quotes were just habit; it doesn't matter here.

I left it that way to show that relative paths work.  `processed` would put the processed files in a directory next to the script.  It also works as `/Volumes/storage/videos/processed`.